src/jit/gentree.h

```diff
@@ -3379,8 +3442,11 @@ struct GenTreeCall final : public GenTree
 
     void ReplaceCallOperand(GenTree** operandUseEdge, GenTree* replacement);
 
+    bool AreArgsComplete();
```

Comment:
Yes.

---
In reply to: [79510728](https://github.com/dotnet/coreclr/pull/7252#discussion_r79510728) [](ancestors = 79510728)